tests/integration/test_k8s_with_cluster.py

```diff
@@ -351,7 +358,8 @@ def test_deployment_with_ssh_github_connectivity(
         assert False
     finally:
         load_kubernetes_config()
-        delete_namespace("bodywork-test-batch-job-project")
+        if namespace_exists("bodywork-test-batch-job-project"):
```

Comment:
Don't we have to delete the SSH secret that will have been created (by default)?

Reply:
This will be updated (it's a valid test), but it should still be deleted.